tests/unittests/api/helpers/test_db.py

```diff
@@ -3,7 +3,7 @@
 from app import current_app as app
 from tests.unittests.utils import OpenEventTestCase
 from app.factories.event import EventFactoryBasic
-from app.api.helpers.db import save_to_db, safe_query
+from app.api.helpers.db import save_to_db, safe_query, get_or_create, get_count
```

Comment:
I didn't get 😅 This test file is present at `tests/unittests/api/helpers` and we are testing those db helpers from API